cardano-cli/test/cardano-cli-golden/Test/Golden/Governance/StakeAddress.hs

```diff
@@ -0,0 +1,98 @@
+{- HLINT ignore "Use camelCase" -}
+
+module Test.Golden.Governance.StakeAddress where
+
+import           Control.Monad (void)
+
+import           Test.Cardano.CLI.Util (execCardanoCLI, noteInputFile, propertyOnce)
+
+import           Hedgehog
+import qualified Hedgehog.Extras.Test.Base as H
+import qualified Hedgehog.Extras.Test.Golden as H
+
+hprop_golden_conway_stakeaddress_delegate_no_confidence :: Property
+hprop_golden_conway_stakeaddress_delegate_no_confidence =
+  propertyOnce . H.moduleWorkspace "tmp" $ \tempDir -> do
+    vkeyFile <- noteInputFile "test/cardano-cli-golden/files/input/conway/stake.vkey"
+    delegFile <- H.noteTempFile tempDir "deleg"
+    delegGold <- H.note "test/cardano-cli-golden/files/golden/governance/stakeaddress/noConfidenceDeleg.cert"
+
+    void $ execCardanoCLI
+      [ "conway", "stake-address", "vote-delegation-certificate"
+      , "--stake-verification-key-file", vkeyFile
+      , "--always-no-confidence"
+      , "--out-file", delegFile
+      ]
+
+    H.diffFileVsGoldenFile delegFile delegGold
```

Comment:
can we check if the output file contains the vote we desire here?

Reply:
We could parse the cbor on the envelope and make sure it ends with 8103. But the test as is will fail if it doesn't.